src/element-style-observer.js

```diff
@@ -229,7 +226,7 @@ export default class ElementStyleObserver {
 		// Only add properties not already present
 		let transition = properties
 			.filter(property => !transitionProperties.has(property))
-			.map(property => `${property} 1ms step-start${allowDiscrete}`)
+			.map(property => `${property} 1ms step-start${this.constructor.getAllowDiscrete()}`)
```

Comment:
This was not a side effect, I'd just add `globalThis?.` before the original const

Reply:
True! Done in https://github.com/LeaVerou/style-observer/pull/111/commits/79fffb301351a90574b886af4f660772ad9511be, thank you.